backend/app/usecase/repository/shortlink_fake.go

```diff
@@ -21,15 +21,24 @@ func (s ShortLinkFake) IsAliasExist(alias string) (bool, error) {
 }
 
 // CreateShortLink inserts a new ShortLink into short_link table.
-func (s *ShortLinkFake) CreateShortLink(shortLink entity.ShortLink) error {
-	isExist, err := s.IsAliasExist(shortLink.Alias)
+func (s *ShortLinkFake) CreateShortLink(shortLinkInput entity.ShortLinkInput) error {
+	if shortLinkInput.CustomAlias == nil {
+		return errors.New("alias empty")
+	}
+	isExist, err := s.IsAliasExist(*shortLinkInput.CustomAlias)
 	if err != nil {
 		return err
 	}
 	if isExist {
 		return errors.New("alias exists")
 	}
-	s.shortLinks[shortLink.Alias] = shortLink
+        customAlias := shortLinkInput.GetCustomAlias("")
```

Comment:
```suggestion
       customAlias := shortLinkInput.GetCustomAlias("")
```

Reply:
Not sure why this was spaced out and wasn't caught by the formatter. Thanks for catching this!